py17track/package.py

```diff
@@ -278,3 +281,11 @@ def __attrs_post_init__(self):
         object.__setattr__(self, "origin_country", COUNTRY_MAP[self.origin_country])
         object.__setattr__(self, "package_type", PACKAGE_TYPE_MAP[self.package_type])
         object.__setattr__(self, "status", PACKAGE_STATUS_MAP[self.status])
+
+        if self.timestamp is not None:
+            timestamp_loc = timezone(self.tz).localize(
+                datetime.strptime(self.timestamp, "%Y-%m-%d %H:%M")
+            )
+            timestamp_utc = timestamp_loc.astimezone(timezone("UTC"))
```

Comment:
You don't need to do this if `tz` is already passed in as `"UTC"`; check for that case.

Additionally, you can save a method call by replacing `timezone("UTC")` with `pytz.UTC` (you'll need to import it).

Reply:
implemented